App/Containers/ThreadsList.js

```diff
@@ -96,6 +96,8 @@ class ThreadsList extends React.PureComponent {
       })
     }
     if (
+      this.props.threads &&
+      prevProps.threads &&
```

Comment:
This was a bug hiding if there are no threads to display and the logic in here says not to display the onboarding content.